.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-13, windows-latest]
```

Comment:
According to https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources (which I suspect you've read), macos-13 is in beta, so I suppose we should switch back to macos-latest once that points to macos-13. I'm not sure a TODO will be effective, so I'll let you decide if we should have one or not :)

Reply:
Yeah, good point, but I can just create a ticket and assign it to myself instead, which would probably be better, I think?

Reply:
Yes, that seems better :)